pkgs/applications/networking/cluster/k3s/default.nix

```diff
@@ -34,6 +34,7 @@ let
     url = "https://kubernetes-charts.storage.googleapis.com/traefik-${traefikChartVersion}.tgz";
     sha256 = "1aqpzgjlvqhil0g3angz94zd4xbl4iq0qmpjcy5aq1xv9qciwdi9";
   };
+  pkgMaintainers = [ maintainers.euank ];
```

Comment:
please don't move this up here - it doesn't save us a single line, and people might want to step up maintaining individual packages from below.

Reply:
Done. My thought process for that was that there really is only one package that matters here (all the intermediate ones are only part of creating the final build), so the set of maintainers for them should always be the same.